kornia/augmentation/_2d/intensity/jpeg.py

```diff
@@ -0,0 +1,55 @@
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+from kornia.augmentation import random_generator as rg
+from kornia.augmentation._2d.intensity.base import IntensityAugmentationBase2D
+from kornia.core import Tensor
+from kornia.enhance import jpeg_codec_differentiable
+
+
+class RandomJPEG(IntensityAugmentationBase2D):
+    r"""Applies random (differentiable) JPEG coding to a tensor image.
+
```

Comment:
Can you add an image example here?

Reply:
Can we use the same image as here? 😅

https://github.com/kornia/kornia/blob/e40da282ba84b02ed8f3958dc220ce76d20572aa/kornia/enhance/jpeg.py#L410

Reply:
ideally we auto generate here https://github.com/kornia/kornia/blob/main/docs/generate_examples.py

Reply:
Just added `RandomJPEG` to the auto generate script and linked the resulting image.